srcpkgs/hamlib/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'hamlib'
+pkgname=hamlib
+version=3.3
+revision=1
+build_style=gnu-configure
+configure_args="--disable-static --with-perl-binding --with-python-binding
+ --with-tcl-binding --with-xml-support"
+pycompile_module="Hamlib.py"
+hostmakedepends="automake libtool pkg-config python3 swig"
+makedepends="gettext-devel libusb-devel libxml2-devel python3-devel readline-devel
+ tcl-devel"
+depends="perl"
+short_desc="Library to control radio transceivers and receivers"
+maintainer="bra1nwave <brainwave@openmailbox.org>"
+license="LGPL-2.1-or-later, GPL-2.0-or-later"
+homepage="https://sourceforge.net/projects/hamlib/"
+distfiles="${SOURCEFORGE_SITE}/hamlib/${version}/${pkgname}-${version}.tar.gz"
+checksum=c90b53949c767f049733b442cd6e0a48648b55d99d4df5ef3f852d985f45e880
+
+pre_configure() {
+	CFLAGS+=" -DNO_POSIX_2008_LOCALE -D_GNU_SOURCE"
```

Comment:
`-DNO_POSIX_2008_LOCALE` is set in the perl template only for musl. Is it perhaps better to use it here also only conditionally?